gateway-service-knoxsso/src/main/java/org/apache/knox/gateway/service/knoxsso/WebSSOResource.java

```diff
@@ -47,6 +50,10 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.knox.gateway.audit.log4j.audit.Log4jAuditor;
 import org.apache.knox.gateway.config.GatewayConfig;
+import org.apache.commons.codec.binary.Base64;
```

Comment:
Since we are on Java 8 minimum, should be able to use Base64 from the JDK and not need commons-codec.

Reply:
Sure, can use the java.util.Base64.